packages/lib-user/src/components/UserHome/components/RecentProjects/RecentProjectsContainer.js

```diff
@@ -1,90 +1,50 @@
 import { shape, string } from 'prop-types'
-import { panoptes } from '@zooniverse/panoptes-js'
-import useSWR from 'swr'
-import auth from 'panoptes-client/lib/auth'
 
-import { usePanoptesProjects } from '@hooks'
+import { usePanoptesProjects, useProjectPreferences } from '@hooks'
```

Comment:
This container used to have a custom function to fetch project prefs, but now there's a shared hook.